.github/workflows/warnings-stable.yml

```diff
@@ -0,0 +1,50 @@
+name: Checking warnings on stable
```

Comment:
On stable what?

Reply:
Branches. I will append `branches` to the name.